packages/babel-plugin-remove-graphql-queries/src/__tests__/index.js

```diff
@@ -9,6 +9,53 @@ function matchesSnapshot(query) {
   expect(code).toMatchSnapshot()
 }
 
+it(`Transforms queries in useStaticQuery`, () => {
+  matchesSnapshot(`
+  import React from 'react'
+  import { graphql, useStaticQuery } from 'gatsby'
+
+  export default () => {
+    const siteTitle = useStaticQuery(graphql\`{site { siteMetadata { title }}}\`)
+    
+    return (
+      <h1>{siteTitle.site.siteMetadata.title}</h1>
+    )
+  }
+  `)
+})
+
+it(`Transforms queries defined in own variable in useStaticQuery`, () => {
+  matchesSnapshot(`
+  import React from 'react'
+  import { graphql, useStaticQuery } from 'gatsby'
+
+  export default () => {
+    const query = graphql\`{site { siteMetadata { title }}}\`
```

Comment:
Generally these are used below/outside of the invoking function. Could we update this snapshot like so?

For example:

```jsx
import React from 'react'
import { graphql, useStaticQuery } from 'gatsby'

export default function Whatever() {
  const data = useStaticQuery(siteQuery)

  return <h1>{data.site.siteMetadata.title}</h1>
}

export const siteQuery = graphql`
  {
    site {
      siteMetadata {
        title
      }
    }
  }
`
```

Reply:
(this will also validate the `export` still works)

Reply:
Added a test for this!